addons/survey/views/survey_templates.xml

```diff
@@ -65,7 +65,7 @@
         <t t-call="survey.layout">
```

Comment:
Looking good, in the end we now have the same layout for all pages it seems.

I just noticed something... :eyes: 

![image](https://github.com/odoo/odoo/assets/41282536/ff664ff6-d74e-41d0-8db0-3fe8f764c971)

Could we get rid of it? It's really badly placed in front of the navigation ^^
(Would add a small note about it in the commit message).

It's strange because it seems done :thinking: 
https://github.com/odoo/odoo/blob/17.0/addons/survey/views/survey_templates.xml#L8